package.json

```diff
@@ -60,7 +60,7 @@
     "@endpass/class": "^0.20.1",
```

Comment:
can't see any declaration in `font-family` of new font

Reply:
here it's declared in `App.vue`

```
body,
input,
button,
select {
  font-family: 'Brutal Type', Helvetica, Arial, sans-serif;
}
```

all remains inherit font-family from `@endpass/ui`. When you will approve, you can check real font-family via devtools.